### PR TITLE
Add Link to doxygen API ref

### PR DESCRIPTION
**Pull request by EduPonz**
Signed-off-by: EduPonz <eduardoponz@eprosima.com>

### CODE REVIEW

File: docs/fastdds/api_reference/api_reference.rst [hunk 1]
```diff
@@ -3,13 +3,16 @@
...
-Fast DDS, as a Data Distribution Service (DDS) standard implementation, exposes the DDS Data-Centric Publish-Subscribe
+*Fast DDS*, as a Data Distribution Service (DDS) standard implementation, exposes the DDS Data-Centric Publish-Subscribe
 (DCPS) Platform Independent Model (PIM) API, as specified in the
 `DDS specification <https://www.omg.org/spec/DDS/1.4/PDF>`_.
 Furthermore, is also gives the user the possibility to directly interact with the underlying Real-time
 Publish-Subscribe (RTPS) API that DDS implements for wired communications, as specified in the
 `RTPS standard <https://www.omg.org/spec/DDSI-RTPS/2.2/PDF>`_.
 
+This section presents the most commonly used APIs provided by *Fast DDS*.
+For a complete API reference, please refer to
```
Comment (JLBuenoLopez-eProsima) [inline docs/fastdds/api_reference/api_reference.rst:14]: As far as I see this API reference only includes the old Fast RTPS PubSub API. Consequently, I suggest saying:

```suggestion
For more information about the API reference, please refer to
```

**Review by JLBuenoLopez-eProsima — COMMENTED** [review]
A nitpick. If you are OK with saying "complete API reference" even if it is not, I will approve.

**Comment by richiprosima** [comment]
Build status [BuildStatus](http://jenkins.eprosima.com:8080/job/fastdds_docs_manual/23/)

**Review by JLBuenoLopez-eProsima — APPROVED** [review]
LGTM

**Pull request merged**